### PR TITLE
[FEATURE] Internship Page

### PR DESCRIPTION
**Pull request by adityayadav71**
1. Completed  #19  
2. Created internship.astro page in pages.
3. Created InternshipHero.astro, CompanyCard.tsx and InternshipCompanies.tsx component in src/Internships directory.
4. Used Array to map CompanyCard components in InternshipCompanies component.
5. Used CountUp component from Website Hero section in InternshipHero component.
6. Added necessary assets in public/InternshipHero directory.

### CODE REVIEW

File: src/components/Internships/InternshipHero.astro [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+import CountUpComponent from "../Hero/CountUpComponent";
+---
+
+<div class="flex lg:flex-row flex-col items-center gap-12 w-full mt-12 mb-12">
+  <div class="w-full lg:order-1 order-2">
+    <div class="mb-6 rounded-xl">
+      <img src="/InternshipHero/img_1.png" class="rounded-xl" alt="image-of-a-female-student-pointing-towards-a-laptop-screen" />
+    </div>
+    <div class="flex sm:flex-row flex-col items-center w-full gap-3">
```
Comment (riyavishwa981) [inline src/components/Internships/InternshipHero.astro:10]: The two boxes are not aligning with the image ty to fix that

Comment (riyavishwa981) [inline src/components/Internships/InternshipHero.astro]: The two count boxes are not aligning with the image ty to fix that

**Review by riyavishwa981 — CHANGES_REQUESTED** [review]

File: src/components/Internships/InternshipHero.astro (hunk 1 above)
Comment (adityayadav71) [reply src/components/Internships/InternshipHero.astro]: Fixed now

**Review by riyavishwa981 — APPROVED** [review]

**Comment by JayeshVP24** [comment]
Hero section is very well done, nice job there
now the companies list part

- [x] 8 rows on 2xl feels very filled. make it 6
- [x] change text color of company name -> `text-slate-600` prefer to use tailwind defaults
- [x] change border below companies offering internship title to `border-slate-200`
- [x] also increase this borders width. it feels too small. make it responsive too. on large screens make the width shorter. and small screens make it longer. Try to match it with figma design

rest all is great. good use of reusable components and other folders components. great job

**Comment by adityayadav71** [comment]
Thanks for reviewing my PR. I've made the requested changes:

- Reduced rows to 6 on 2xl screens
- Changed text color of company name to text-slate-600
- Changed border color and increased width below companies offering internship title
- Made the border responsive as per the Figma design
Appreciate your feedback and I am looking forward to your further review.

**Pull request merged**